legend-engine-language-pure-modelManager-sdlc/src/main/java/org/finos/legend/engine/language/pure/modelManager/sdlc/SDLCLoader.java

```diff
@@ -180,7 +180,9 @@ else if (context.sdlcInfo instanceof AlloySDLC)
                     }
                     else
                     {
-                        throw new RuntimeException("The following PackageableElementPointers:" + missingPaths.toString() + " do not exist in the project data loaded from the metadata server");
+                        throw new RuntimeException("The following PackageableElementPointers:" + missingPaths + " do not exist in the project data loaded from the metadata server. " +
+                                "Please make sure the corresponding Gitlab pipeline for version " + (this.alloyLoader.isLatestRevision(sdlc) ? "latest" : sdlc.version) + " has completed and also metaData server has updated with corresponding entities " +
+                                "by confirming the data returned from <a href=\"" + this.alloyLoader.getMetaDataApiUrl(pm, sdlc, clientVersion) + "\"/> this Api </a>.");
```

Comment:
Might be worth clarifying what we mean by confirming the data returned - I guess we want to say "confirm that data is returned" and the that response is not empty?

Also change Api to API?

Reply:
Hi Aziem, I mentioned `metadata server has updated with corresponding entities` in front of `..data..`. Will it be more clearer after this explaination?

Reply:
ok I see what you mean. Yes this makes sense looking at the whole sentence